pages/blueprint/index.js

```diff
@@ -278,6 +280,34 @@ class BlueprintPage extends React.Component {
     return `/cockpit/channel/${cockpit.transport.csrf_token}?${query}`;
   }
 
+  imageTypeToLabel(imageType) {
+    const imageTypeLabels = {
+      ami: "Amazon Machine Image Disk (.ami)",
```

Comment:
The issue of `.ami` is that the file extension was actually made up by us. AFAIK it isn't mentioned anywhere in AWS docs and also the produced image doesn't have this extension - we use an extension that represents the actual file's format. In the past, it was `.vhdx`, now it's `.raw` and I hope we will switch it to `.vmdk` soon.

Reply:
@jkozol What are your thoughts on updating this in this PR?

Reply:
@jgiardino I am fine including this in this PR. Do you have any thoughts on what this image type should be named? "Amazon Raw Image"?

Reply:
Oooh, thanks for asking :-D

Below is a screencap of what we used in the usability prototype. This was something I reviewed with @andreasn, and the consensus was that the labels should reflect the name of the platform or service provider. So for AWS, we replaced "Amazon Machine Image Disk" with "Amazon Web Services". It would be awesome if these labels could be updated based on what's below.

<img width="391" alt="image" src="https://user-images.githubusercontent.com/21063328/88670570-39765080-d0b3-11ea-80df-ec6d71b4830d.png">

Providing the correct file type is tricky though. Ideally, this is provided by the API, since the API is defining what file type to generate. And since it sounds like it is likely to change, it would likely be incorrect at some point in time (i.e. now) if defined in the UI. I expect many users would want to know what file type they would get when selecting an image type before kicking off a process that could take some time (and some users would probably want this info  when researching the tool to determine if it's something they would want to use). I understand the argument that the file type isn't important in the context of the complete image upload workflow for cloud images, but that's only a portion of the workflows supported. So I'm on the fence about this one. Does anyone have thoughts on whether we should pull them or keep and maintain them as they change?

Reply:
I updated the image labels to these without the section headers. I can do that in a seperate PR if you would like. I also think we should maybe pass more info from the api's /compose/types route in order to get the both the image type and image label (i.e. Azure vs. .vhd) since there are multiple providers for the same image type (i.e. qcow2).

Reply:
👍 

> I updated the image labels to these without the section headers. I can do that in a seperate PR if you would like.

I think this is another one of those things we need from the API (cc'ing @teg on this), so I wouldn't add any groupings until that's provided.

Reply:
@jgiardino ok. I believe that the groupings are outside the scope of this PR since the goal here was just to standard the image list and upload list with the labels used in the wizard. Is it okay if we move this to an RFE issue?

Reply:
Yep, that's what I was thinking, and what I meant to suggest. 🙃